roofit/roofit/inc/RooPyBindFunction.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   Robin Syring, CERN 2024
+ *
+ * Copyright (c) 2023, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#ifndef ROOPYBINDFUNCTION_H
+#define ROOPYBINDFUNCTION_H
+
+#include "RooAbsReal.h"
+#include "RooListProxy.h"
+#include "RooArgList.h"
+
```

Comment:
```suggestion

namespace RooFit {
namespace Detail {

...
```
Can you maybe put the class inside the `RooFit::Detail` namespace? That signifies to the user that the class should not be used directly, but it's a RooFit implementation detail.